### ISSUE TITLE
Standards list switcher does not work in IE8

### ISSUE THREAD

**Issue by luketw** [squizlabs/HTML_CodeSniffer#82]
When the Section 508 standards were added, the standards list population was changed to handle more than just the WCAG2\* standards. However, in IE8 the standards list is empty.

Standards (and sniffs) are declared as vars in the global JS namespace, seems to make them properties of the Window object in Firefox and Chrome, but in IE8 it doesn't. Since it depends on "for i in window" type iteration in the auditor, it is returning no standards. Referring to standards with the "window." prefix works however.

Workaround is currently to modify the bookmarklet to use the appropriate standard code. Will work out a more robust method of including the standards, probably by making them a property inside the HTMLCS namespace.
